### ISSUE TITLE
Using 0 as default for coerce number returns undefined

### ISSUE THREAD

**Issue by jlimas** [hapipal/confidence#115]
As the title says, using 0 as the $default value for a $coerce number returns undefined.

Example key in store.

```
messages: {
    delay: {
        $env: 'MESSAGES__DELAY',
        $coerce: 'number',
        $default: 0,
    }
}
```

The test case

```
it('should coerce values to number', () => {
    expect(store.get('/messages/delay')).toBe(0)
})
```

![image](https://user-images.githubusercontent.com/368854/154897861-5740a244-b391-47fd-903b-c8edf5bfad40.png)

Using any other value for $default like 1 makes the test pass without problems.

**Comment by Nargonath** [comment]
This looks like a bug to me

**Comment by devinivy** [comment]
Fix from #116 published in v6.0.2, thanks again!

**Comment by Nargonath** [comment]
Thanks for the quick fix @jlimas.
